blog/announcements.xml

```diff
@@ -2,6 +2,115 @@
 
 <news>
 
+  <item>
+    <pubDate>Tue, 30 Nov 2021 18:00:00 UTC</pubDate>
+    <title>
+      <a id="21.11">NixOS 21.11 released</a>
+    </title>
+    <description>
+      <a href="https://github.com/NixOS/nixos-artwork/blob/master/releases/21.11-porcupine/porcupine.png">
+        <img class="inline" src="/logo/nixos-logo-21.11-porcupine-lores.png" alt="21.11 Porcupine logo" width="100" />
+      </a>
+      <p>
+        Hey everyone, we're <a href="https://github.com/nrdxp">Timothy DeHerrera</a> and <a href="https://github.com/tomberek">Tom Bereknyei</a>,
+        the release managers for 21.11. As promised, the latest stable
+        release is here: NixOS 21.11 “Porcupine”.
+      </p>
+
+      <ul>
+        <li>
+          <a href="/manual/nixos/stable/release-notes.html#sec-release-21.11">Release manual</a>
+          <ul>
+            <li><a
+                href="/manual/nixos/stable/release-notes.html#sec-release-21.11-highlights">Highlights</a>
+            </li>
+            <li><a href="/manual/nixos/stable/release-notes.html#sec-release-21.11-new-services">New
+                Services</a></li>
+            <li><a
+                href="/manual/nixos/stable/release-notes.html#sec-release-21.11-incompatibilities">Backward
+                Incompatibilities</a></li>
+            <li><a
+                href="/manual/nixos/stable/release-notes.html#sec-release-21.11-notable-changes">Other
+                Notable Changes</a></li>
+          </ul>
+        </li>
+        <li><a href="/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="/download.html#download-nixos">Image downloads</a></li>
+      </ul>
+
+      <p>
+        The 21.11 release was possible due to the efforts of <strong>1541</strong>
+        contributors in <strong>41960</strong> commits. We would especially
+        like to thank our top 10 contributors:
+        <a href="https://github.com/supersandro2000">Sandro Jäckel</a>,
+        <a href="https://github.com/fabaff">Fabian Affolter</a>,
+        <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        <a href="https://github.com/figsoda">figsoda</a>,
+        <a href="https://github.com/Artturin">Artturin</a>,
+        <a href="https://github.com/marsam">Mario Rodas</a>,
+        <a href="https://github.com/bobby285271">Bobby Rong</a>,
+        <a href="https://github.com/mic92">Jörg Thalheim</a>,
+        <a href="https://github.com/dotlambda">Robert Schütz</a>,
+        <a href="https://github.com/primeos">Michael Weiss</a>.
+      </p>
```

Comment:
Unclear if this needs to be changed (haven't verified with the previous release)

Reply:
same as above:
https://github.com/NixOS/nixos-homepage/pull/764#discussion_r759724747